### ISSUE TITLE
OPTION の使い方と要望

### ISSUE THREAD

**Issue by muooon** [hako-mikan/sd-webui-traintrain#43]
Option について教えてください

train lr scheduler settings
　CosineAnnealingWarmRestarts のとき、"eta_min" の指定がありました、ということは、この空白欄は、https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.CosineAnnealingWarmRestarts.html にある Parameters を指定できる、のでしょうか。

train optimizer settings 
　https://pytorch.org/docs/stable/optim.html にある EMA / SWA などの設定ができるのでしょうか ？ 

上記２点、他にもどのような指定が可能ですか ？ いろいろと詳しい設定を可能としているようなので教えて頂けると嬉しいです。

要望したいものは、
　｢Latent キャッシュ｣ ｢Debias Estimation loss / ZtSNR｣ ｢Scale Weight Norms｣ の実装です。そして層別学習のチェックボックスについて、XL と SD15 モード の切り替えをできると、とても便利になると思っています。
